src/main/java/com/example/SpringCRUD/controller/EmployeeController.java

```diff
@@ -0,0 +1,65 @@
+package com.example.SpringCRUD.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.SpringCRUD.Repository.EmployeeRepository;
+import com.example.SpringCRUD.entity.EmployeeEntity;
+import com.example.SpringCRUD.exception.RecordNotFoundException;
+import com.example.SpringCRUD.service.EmployeeService;
+
+//import com.accion.cision.service.EmployeeService;
+
+@RestController
+@RequestMapping("/cision")
+public class EmployeeController {
+
+	@Autowired
+	EmployeeService service;
```

Comment:
Give proper naming convention and also provide access modifier

Reply:
Done